cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs

```diff
@@ -121,7 +121,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("coretime-rococo"),
 	impl_name: create_runtime_str!("coretime-rococo"),
 	authoring_version: 1,
-	spec_version: 1_005_002,
+	spec_version: 1_005_003,
```

Comment:
This was done because we needed to debug some XCM message on the coretime chain.